libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -48,6 +47,8 @@
   private int currentManeuverId;
   private boolean isTwentyFourHourFormat;
   private String etaFormat;
+  private final Context context;
```

Comment:
Minor comment - to note that should be an application context, what about calling it `applicationContext` instead? As in https://github.com/mapbox/mapbox-navigation-android/blob/5eabb77d9b5cd061fccf96d3f6f49ce1bc5501c1/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java#L50